bin/opm

```diff
@@ -1509,7 +1510,14 @@ SKIP_INSTALL:
 }
 
 sub get_install_dir () {
-    if (defined $install_into_cwd) {
+    if (defined $install_into_dir) {
```

Comment:
I think we need a check for the case that both `--install-dir=PATH` and `--cwd` are specified. We should generate an error instead of assuming the former overrides the latter.